packages/compass-workspaces/src/provider.tsx

```diff
@@ -65,7 +65,7 @@ export type WorkspacesService = {
     options?: TabOptions &
       Omit<
         Extract<OpenWorkspaceOptions, { type: 'Collection' }>,
-        'type' | 'namespace' | 'editViewName'
+        'type' | 'namespace' | 'editViewName' | 'subTab'
```

Comment:
For the openWorkspaceOptions this should probably be initialSubTab and should be optionally allowed